src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -105,6 +108,7 @@ class ApplicationMenu extends Component<Props> {
 const mapDispatchToProps = {
   createNewProjectStart,
   showImportExistingProjectPrompt,
+  clearConsole,
```

Comment:
So I think if we aren't adding ApplicationMenu support, we should remove these changes for now. It can be added in a subsequent PR, but I don't know where that is in the timeline, so it feels safer to omit it for now :)